integrationtest/helper.go

```diff
@@ -0,0 +1,163 @@
+package integrationtest
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/taskcluster/slugid-go/slugid"
+	tcclient "github.com/taskcluster/taskcluster-client-go"
+	"github.com/taskcluster/taskcluster-client-go/queue"
+	"github.com/taskcluster/taskcluster-worker/commands"
+	_ "github.com/taskcluster/taskcluster-worker/commands/work"
+	_ "github.com/taskcluster/taskcluster-worker/config/env"
+	_ "github.com/taskcluster/taskcluster-worker/config/secrets"
+	_ "github.com/taskcluster/taskcluster-worker/engines/native"
+)
+
+var (
+	// all tests can share taskGroupId so we can view all test tasks in same
+	// graph later for troubleshooting
+	taskGroupID string = slugid.Nice()
+	testdata    string = testdataDir()
+)
+
+func testdataDir() string {
+	// some basic setup...
+	cwd, err := os.Getwd()
+	if err != nil {
+		panic(fmt.Errorf("Could not get current directory during test package initialisation: %v", err))
```

Comment:
This is executed during package initialisation, not during a test, so we have to panic - no test is running at this point.